src/app.rs

```diff
@@ -14,7 +14,20 @@ pub(crate) struct App {
   worker: Worker,
 }
 
-const EXAMPLES: include_dir::Dir = include_dir!("examples");
+lazy_static! {
+  static ref EXAMPLES: HashMap<&'static str, &'static str> = {
+    let mut m = HashMap::new();
+    m.insert("All", include_str!("../examples/all.js"));
+    m.insert("Kaleidoscope", include_str!("../examples/kaleidoscope.js"));
+    m.insert("Orb Zoom", include_str!("../examples/orb_zoom.js"));
+    m.insert("Orbs", include_str!("../examples/orbs.js"));
+    m.insert("Pattern", include_str!("../examples/pattern.js"));
+    m.insert("Starburst", include_str!("../examples/starburst.js"));
+    m.insert("Target", include_str!("../examples/target.js"));
+    m.insert("X", include_str!("../examples/x.js"));
+    m
+  };
```

Comment:
You should be able to do something like this:

```suggestion
  static ref EXAMPLES: HashMap<&'static str, &'static str> = [
    ("All", include_str!("../examples/all.js")),
    ("Kaleidoscope", include_str!("../examples/kaleidoscope.js")),
    ("Orb Zoom", include_str!("../examples/orb_zoom.js")),
    ("Orbs", include_str!("../examples/orbs.js")),
    ("Pattern", include_str!("../examples/pattern.js")),
    ("Starburst", include_str!("../examples/starburst.js")),
    ("Target", include_str!("../examples/target.js")),
    ("X", include_str!("../examples/x.js")),
  ].iter().collect();
```

You might have to add a `.cloned()` or something to make the types work.